english/eating-psychadelics/eating-psychadelics.md

```diff
@@ -0,0 +1,9 @@
+You recall the previous night and how you played some sort of version of beer pong but the cups where filled with a golden syrup sort of liquor.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.